main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: sun 
+//  action 1: the sun shines
+sun.shine();
+//  action 2:    the sun burns
+sun.burn();
+//  action 3:    the sun lights up
+sun.lightUp();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: hand   
+//  action 1: the hand opens
+hand.open();
+//  action 2: the hand closes
+hand.close();
+//  action 3: the hand shake
+hand.shake();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: dog
+//  action 1: the dog wags its tail
+dog.wag();
+//  action 2: the dog barks
+dog.bark();
+//  action 3: the dog plays
+dog.play();
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: bee
+//  action 1: bee flies
+bee.fly();
+//  action 2: bee collects the nectar from plants
+bee.collectNectar();
+//  action 3: bee produces wax
+bee.produceWax();

```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.